### DISCUSSION TITLE
Build and Deploy Job Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20:

### DISCUSSION THREAD

**Discussion by galilynnn** [microsoft/SATechnicalOnboarding#141] [Questions]
I followed the instructions step-by-step but keep encountering an error with my build. My Node.js version is v20.11.1. I also tried running it on GitHub Codespaces, but nothing works.

edit1: I tried redo the project over 10 times, still keep getting the same error
Here is a screenshot of the error:
<img width="549" alt="Screenshot 2567-06-14 at 12 31 43" src="https://github.com/microsoft/SATechnicalOnboarding/assets/108887565/c3e828f1-7d3b-4fd9-8a13-e264cbcc62ab">

**Comment by AnthonyByansi** [comment]
Hi @galilynnn 
You should update the package.json file with your personal details. 
![github-pages-deploy](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/62fe10bc-6fca-473c-9d8b-7bf76c14e62b)

After editing the configurations in the `package.json` file, everything should operate smoothly. I hope that helps!
